site/en/docs/web-platform/origin-trials/index.md

```diff
@@ -138,8 +137,22 @@ registration and provide a new token, for each origin enrolled in the trial.
 3. Copy the new token and make it available for every page that should remain enrolled in the trial.
 You can provide multiple tokens if necessary: Chrome will ignore invalid or expired tokens.
 
+{% Aside %}
+
+Usually if an API lands unchanged after a successful origin trial, there is a
+short period between the end of the origin trial and the date the implementation ships in the
+browser when the API is unavailable. This is by design. If Chrome were to avoid the
+mandatory total-breakage period, that would bias toward avoiding breakages in the API surface. Such
+breakages are often needed to improve the API. The final shipping API might be worse for it.
```

Comment:
"If Chrome were to avoid the mandatory total-breakage period, that would bias toward avoiding breakages in the API surface."

Who is the owner of "bias"? I'm a bit confused by the sentence in context.

Would this be correct:

```suggestion
Usually if an API lands unchanged after a successful origin trial, there is a
short period between the end of the origin trial and the date the implementation
ships in the browser when the API is unavailable. This is intentional and known
as the total-breakage period. Chrome allows an opportunity for breakage in the
API surface, which is often needed to improve the API. The final shipping API
may be worse without this period.
```

Reply:
I agree this is a bit hard to understand!

I like your rephrasing. @jpchase Does Alexandra's version sound OK to you here?

Reply:
The requirement for a breakage period has actually been [removed](https://www.chromium.org/blink/launching-features/#step-5-optional-origin-trial:~:text=%5BNote%3A%20the-,required%20breaking%20period,-was%20removed%20in). We're ~9 months into the 1 year temporary removal, so it seems likely to become permanent.

I suggest removing the section about breakage entirely. If they do decide to reinstate, we can always add something back.

Reply:
Thanks @jpchase! I've updated the aside to explain the change (with links).